pom.xml

```diff
@@ -0,0 +1,183 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.disq-bio</groupId>
+    <artifactId>disq</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+
+    <properties>
+        <maven.compiler.target>1.8</maven.compiler.target>
+        <maven.compiler.source>1.8</maven.compiler.source>
+    </properties>
+
+    <dependencies>
```

Comment:
Dependency versions are better specified in a separate `<dependencyManagement>` section.  I also like to define version properties so that dependencies from the same groupId with the same version can be updated in one place.

Reply:
Is a `<dependencyManagement>` section useful if there are no child POMs? For the version numbers I've pulled out version properties for htsjdk, Hadoop, and Spark.